ports/clannad/Clannad.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+source $controlfolder/device_info.txt
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+get_controls
+
+# Set variables
+GAMEDIR="/$directory/ports/clannad"
+DEVICE_ARCH="${DEVICE_ARCH:-aarch64}"
+runtime="rlvm"
+rlvm_dir="$HOME/rlvm"
+rlvm_file="$controlfolder/libs/${runtime}.${DEVICE_ARCH}.squashfs"
```

Comment:
The runtime file will be `rlvm.squashfs`, not `rlvm.aarch64.squashfs`.
```suggestion
rlvm_file="$controlfolder/libs/${runtime}.squashfs"
```